Generation/data/Pythia_minBias.cmd

```diff
@@ -0,0 +1,65 @@
+!
+! File: Pythia_standard.cmd
```

Comment:
Maybe we should also add CM energy + colliding particles to the file-title? E.g. `minbias_pp_10TeV`